Packs/AWS-EC2Extension/pack_metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "AWS - EC2 Extension",
+    "description": "This content pack extends the existing AWS EC2 integration by adding the revoke-security-group-egress command. The intent is for this command to be merged into the system AWS EC2 integration.",
```

Comment:
Looks good, but it does not include what this command does. I suggest we change it like this:
"This content pack extends the existing AWS EC2 integration by adding the revoke-security-group-egress command, which removes specified egress rules from a security group for EC2-VPC. The intent is for this command to be merged into the system AWS EC2 integration.",